### PR TITLE
[CI] Test proxy version bump

### PR DESCRIPTION
**Pull request by pvaneck**
This file will be deleted once `eng/common` is updated with the new test proxy, then we'll go back to the automatic updates.

### CODE REVIEW

File: eng/target_proxy_version.txt [hunk 1]
```diff
@@ -1 +1 @@
-1.0.0-dev.20240506.5
+1.0.0-dev.20240513.2
```
Comment (scbedd) [inline eng/target_proxy_version.txt:1]:
```suggestion
1.0.0-dev.20240515.4
```

Comment (scbedd) [reply eng/target_proxy_version.txt:1]: 1 PR turned into ~3 😆

**Comment by pvaneck** [comment]
Closing this in favor of: https://github.com/Azure/azure-sdk-for-python/pull/35694

**Pull request closed without merging**